R/service-navigation.R

```diff
@@ -0,0 +1,138 @@
+#' Service navigation
+#'
+#' @description
+#' Service navigation component consistent with the
+#' [GDS service navigation](https://design-system.service.gov.uk/components/service-navigation/)
+#' (pre-June 2025).
```

Comment:
I'd take that time reference out as it's easy to miss when updating and could go out of date - rest of the package is still mostly pre-June 2025 while we're waiting on the next major release, so I think fine to leave as assumed

```suggestion
#' [GDS service navigation](https://design-system.service.gov.uk/components/service-navigation/).
```